pkg/apis/config/v1alpha2/meshrootcertificate.go

```diff
@@ -36,6 +36,10 @@ type MeshRootCertificateSpec struct {
 
 	// Intent of the MeshRootCertificate resource
 	Intent MeshRootCertificateIntent `json:"intent"`
+
+	// SpiffeEnabled will add a SPIFFE ID to the certificates, creating a SPIFFE compatible x509 SVID document
+	// To use SPIFFE ID for validation and routing,  'enableSPIFFEValidation' must be true in the Mesh Config after the certificate is made 'active'
```

Comment:
Update certificate to meshRootCertificate for clarity
```suggestion
	// To use SPIFFE ID for validation and routing, 'enableSPIFFE' must be true in the MeshConfig after the MeshRootCertificate is made 'active'
```